.ko.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# .ko.yaml (repo root)
+defaultBaseImage: cgr.dev/chainguard/static:latest
+platforms: [linux/amd64, linux/arm64]
+
+ldflags:
+  - "-s -w"
+  - "-X main.version={{.Env.VERSION}} -X main.commit={{.Env.GIT_COMMIT}} -X main.date={{.Env.BUILD_DATE}}"
+
+env: [CGO_ENABLED=0]
+
+builds:
+
+  - id: fault-quarantine-module
+    dir: fault-quarantine-module
+    main: .
+
+  - id: fault-remediation
+    dir: fault-remediation-module
+    main: .
+
+  - id: health-events-analyzer
+    dir: health-events-analyzer
+    main: .
+    
+  - id: csp-health-monitor
+    dir: health-monitors/csp-health-monitor
+    main: cmd/csp-health-monitor
+
+  - id: maintenance-notifier
+    dir: health-monitors/csp-health-monitor
+    main: cmd/maintenance-notifier
+
+  - id: syslog-health-monitor
+    baseImage: cgr.dev/chainguard/glibc-dynamic:latest
```

Comment:
does this include the systemd dev pacakge? If not the container will not boot up.

Reply:
I think the `-dev` version of that image does include systemd dev package, it at least package manager that could install. Probably better to fall back to the current `nvcr.io/nvidia/cuda` on this one.